examples/limitless/main.tf

```diff
@@ -25,66 +31,65 @@ locals {
 module "aurora" {
   source = "../../"
 
-  name                        = local.name
-  engine                      = "aurora-postgresql"
-  engine_version              = "16.6-limitless"
-  master_username             = "root"
-  storage_type                = "aurora-iopt1"
-  cluster_monitoring_interval = 30
-  cluster_scalability_type    = "limitless"
+  name           = local.name
+  engine         = "aurora-postgresql"
+  engine_version = "16.9-limitless"
+  storage_type   = "aurora-iopt1"
 
+  cluster_scalability_type    = "limitless"
+  cluster_monitoring_interval = 30
   # https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/limitless-reqs-limits.html
   cluster_performance_insights_enabled          = true
   cluster_performance_insights_retention_period = 31
 
-  create_shard_group        = true
-  compute_redundancy        = 0
-  db_shard_group_identifier = local.name
-  max_acu                   = 16
+  shard_group = {
+    compute_redundancy = 0
+    identifier         = local.name
+    max_acu            = 16
+  }
 
   # aurora limitless clusters do not support managed master user password
-  manage_master_user_password = false
```

Comment:
I think we need to keep this since `manage_master_user_password` defaults to `true`.

Reply:
good catch  - added in https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/pull/525/commits/ee41836e68e424956e70b641bc107487b4b4aecc